include/onnxruntime/core/providers/tensorrt/tensorrt_provider_options.h

```diff
@@ -49,6 +49,7 @@ struct OrtTensorRTProviderOptionsV2 {
   const char* trt_profile_max_shapes{nullptr};           // Specify the range of the input shapes to build the engine with
   const char* trt_profile_opt_shapes{nullptr};           // Specify the range of the input shapes to build the engine with
   int trt_cuda_graph_enable{0};                          // Enable CUDA graph in ORT TRT
+  const char* trt_preview_features{nullptr};             // Trt preview features, features should be separated by semicolon
```

Comment:
The comment says using semicolon to separate string, but the code is actually using comma.
Change the comment here to use comma?

Reply:
Good catch. Updated the comment.